specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/assessments.json

```diff
@@ -278,12 +278,20 @@
       "type": "object",
       "description": "Data regarding 3rd party partner integration",
       "properties": {
+        "partnerName": {
+          "type": "string",
+          "description": "Name of the company of the partner"
+        },
         "secret": {
           "type": "string",
           "description": "secret to authenticate the partner - write only",
           "x-ms-secret": true
         }
-      }
+      },
+      "required": [
+        "partnerName",
+        "secret"
+      ]
```

Comment:
You are making 2 properties required in a stable API version, which is a breaking change. Can you explain?

Reply:
these are changes that were left out in the initial review (completed 2 days ago) and describe the API better
if the partnersData object is present, you must have these properties, otherwise, he request will fail

Reply:
Ok sounds good.